tests/openassetio/test_traits.py

```diff
@@ -12,15 +12,15 @@
 
 class Test_Specification_traitIds:
     def test_when_has_no_traits_returns_empty_list(self):
-        empty_specification = specification.Specification([])
-        assert empty_specification.traitIds() == []
+        empty_specification = specification.Specification(set())
+        assert empty_specification.traitIds() == set()
 
     def test_when_has_traits_returns_expected_trait_ids(self):
-        expected_ids = ["a", "b", "🐠🐟🐠🐟"]
+        expected_ids = {"a", "b", "🐠🐟🐠🐟"}
         populated_specification = specification.Specification(expected_ids)
         ## TODO: Should we use set instead of vector for TraitIds, as
```

Comment:
[fix] Can remove this TODO now :)

Reply:
[Gone](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/305/commits/8ed138c9e80b376c6c74f7c89e6095226af094a9#diff-29adf34bb4665be39936d3cbfcf6d37b4659d84d8fcb4a218f49fe75151079ffL21) :) well remembered, thanks!